haskell/src/Ligo/Util.hs

```diff
@@ -40,3 +43,6 @@ fetchValue rawPath = do
         either (error . pretty) fromVal $ readValue "" valueLiteral
       |]
 
+
+aesonOptions :: A.Options
+aesonOptions = (A.aesonPrefix A.camelCase) { A.omitNothingFields = True}
```

Comment:
This seems exactly the same as the `aesonOptions` defined in `Morley.Metadata.Util.Aeson`.

Why not import and use that one?

Reply:
Make sense, I've removed this and use the one from `Moley.Metadata` instead.